src/System.IO.Compression.Brotli/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,23 @@
+<!-- Not a bug. clrcompression.dll is a native components that ship as part of the package. -->
```

Comment:
Why do need just the Brotli stuff here? What makes the analyzer to shut up for the existing ZLib stuff? Should both be on the same plan?

Reply:
The Analyzer only cares about the exports that are actually used in the library. System.IO.Compression has its own copy of this file that only lists the zlib exports.

Reply:
We could technically make it common and list all of the exports, but I preferred the separation and explicitness of each having their own file.

Reply:
Thanks for the explanation. Separate files make sense.